community/cloud-foundation/templates/ssl_certificate/examples/ssl_certificate.yaml

```diff
@@ -0,0 +1,59 @@
+# Example of the SSL certificate template usage.
+
```

Comment:
Add:

    #
    # Replace the following placeholders with valid values:
    #   <FIXME:certificateContents>: contents of the certificate file
    #   <FIXME:privateKeyContents>: contents of the private key file

Reply:
Fixed.